tests/test_chain.rs

```diff
@@ -4,15 +4,6 @@ use bytes::{Buf, BufMut, Bytes};
 #[cfg(feature = "std")]
 use std::io::IoSlice;
 
-#[test]
-fn collect_two_bufs() {
-    let a = Bytes::from(&b"hello"[..]);
-    let b = Bytes::from(&b"world"[..]);
-
-    let res = a.chain(b).copy_to_bytes(10);
-    assert_eq!(res, &b"helloworld"[..]);
-}
```

Comment:
Even if this is covered by some of the new complicated tests, I'd still like to keep the test. There are advantages to having simple tests.